packages/components/src/presentation-components/properties/inputs/UniquePropertyValuesSelector.tsx

```diff
@@ -160,6 +162,7 @@ function useUniquePropertyValuesRuleset(descriptorRuleset?: Ruleset, field?: Fie
             specifications: [
               {
                 specType: "SelectedNodeInstances",
+                acceptableClassNames: selectedClasses && selectedClasses.map(({ name }) => name.split(":")[1]),
```

Comment:
I believe you should also specify the [acceptablePolymorphically](https://www.itwinjs.org/presentation/content/selectednodeinstances/#attribute-acceptablepolymorphically) attribute.

Reply:
As I understand now that we provide the exact class name there should not be any need for `acceptablePolymorphically`. I added it with the default false flag, unless you have an idea why we would need it.

Reply:
As @saskliutas mentioned in the other comment, we want to handle the classes polymorphically - this is generally how people expect classes to work, unless explicitly specifying that they're not going to be handled polymorphically. I don't see a reason we shouldn't do that.

Reply:
Seems I was under the wrong impression about how the rule would handle child classes in this case, changed to True.